src/components/Patients/PatientVitalModalCreate.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState, useEffect } from "react";
+import {
+  Container, Segment, Icon, Modal, Button,
+} from "semantic-ui-react";
+import * as Yup from "yup";
+import { Form } from "formik-semantic-ui-react";
+import { Formik } from "formik";
+import "react-toastify/dist/ReactToastify.css";
+import styled from "styled-components";
+
+import { getVitals } from "api/vitals";
+import InputField from "components/InputField";
+import InputSelect from "components/InputSelect";
+
+const StyledContainer = styled(Container)`
+  && {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-bottom: 2rem;
+  }
+`;
+
+const StyledTopContainer = styled.div`
+  display: flex;
+  align-items: start;
+  justify-content: center;
+  flex-direction: row;
+  gap: 5rem;
+
+`;
+
+const ButtonGroup = styled.div`
+  padding-top: 2rem;
+  padding-bottom: 1.5rem;
+  float: right;
+`;
+
+const TopInfo = styled.div`
+  display: flex;
+  align-items: flex-start;
+  flex-direction: row;
+  gap: 2rem;
+  width: 100%;
+`;
+
+const initialValues = {
+  vital: "",
+  value: "",
+};
+
+const PatientVitalModalCreate = ({ show, handleClick, handleCreate }) => {
+  const [loading, setLoading] = useState();
+  const [error, setError] = useState();
+  const [vitals, setVitals] = useState([]);
+  let vitalOptions;
+  let vitalValues;
+  let validationSchema;
+
+  useEffect(() => {
+    const fetch = async () => {
```

Comment:
Please rename it because of https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch